pkgs/misc/vim-plugins/update.py

```diff
@@ -292,7 +292,7 @@ def generate_nix(plugins: List[Tuple[str, Plugin]]):
     name = "{plugin.normalized_name}-{plugin.version}";
     src = fetchFromGitHub {{
       owner = "{owner}";
-      repo = "{plugin.name}";
+      repo = "{repo}";
```

Comment:
oh, snap. I wonder why I was able to build all plugins. I blame fixed-input derivations for that.